unfoldingWord/translationCore
# Fixing tests

#### This pull request addresses:

- Fixed ulb v11 failing tests


#### How to test this pull request:

- Make sure all tests pass.


---
This change is [<img src="https://reviewable.io/review_button.svg" height="34" align="absmiddle" alt="Reviewable"/>](https://reviewable.io/reviews/unfoldingword-dev/translationcore/2874)
